docs/metrics/exemplars/README.md

```diff
@@ -1,99 +1,154 @@
 # Using Exemplars in OpenTelemetry .NET
 
+- [Install and run Jaeger](#install-and-run-jaeger)
+- [Install and run Prometheus](#install-and-run-prometheus)
+- [Install and configure Grafana](#install-and-configure-grafana)
+- [Export metrics and traces from the
+  application](#export-metrics-and-traces-from-the-application)
+- [Use exemplars to navigate from metrics to
+  traces](#use-exemplars-to-navigate-from-metrics-to-traces)
+- [Learn more](#learn-more)
+
 Exemplars are example data points for aggregated data. They provide specific
 context to otherwise general aggregations. One common use case is to gain
 ability to correlate metrics to traces (and logs). While OpenTelemetry .NET
 supports Exemplars, it is only useful if the telemetry backend also supports the
-capabilities. This tutorial uses well known open source backends to demonstrate
-the concept. The following are the components involved:
+capabilities. This tutorial uses well known open-source backends to demonstrate
+the concept. The following components are involved:
 
-* Test App - We use existing example app from the repo. This app is already
-instrumented with OpenTelemetry for logs, metrics and traces, and is configured
-to export them to the configured OTLP end point.
-* OpenTelemetry Collector - An instance of collector is run, which receives
-telemetry from the above app using OTLP. The collector then exports metrics to
-Prometheus, traces to Tempo.
-* Prometheus - Prometheus is used as the Metric backend.
-* Tempo - Tempo is used as the Tracing backend.
-* Grafana - UI to query metrics from Prometheus, traces from Tempo, and to
-  navigate between metrics and traces using Exemplar.
+- [Program.cs](./Program.cs) - this application is instrumented with
+  OpenTelemetry, it sends metrics to Prometheus, and traces to Jaeger.
+- [Prometheus](#install-and-run-prometheus) - Prometheus is used as the metrics
+  backend.
+- [Jaeger](#install-and-run-jaeger) - Jaeger is used as the distributed tracing
+  backend.
+- [Grafana](#install-and-configure-grafana) - UI to query metrics from
+  Prometheus, traces from Jaeger, and to navigate between metrics and traces
+  using Exemplars.
 
-All these components except the test app require additional configuration to
-enable Exemplar feature. To make it easy for users, these components are
-pre-configured to enable Exemplars, and a docker compose file is provided to
- spun them all up, in the required configurations.
+## Install and run Jaeger
```

Comment:
I wouldn't suggest removing the docker-compose solution. There are a lot of manual steps involved without that. Please keep that for those who want to get up and running by executing a single docker compose.

Reply:
keep the compose option, but need to modify it according to this PR where we use Jaeger and removed collector.

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet/pull/5636#discussion_r1608584627, I feel the docker-compose instructions would fit better in opentelemetry-demo.

Reply:
@cijothomas let me know if you strongly disagree, my main intention of this PR is to remove Docker dependency.

Reply:
> my main intention of this PR is to remove Docker dependency.

I don't get the reasoning behind that. What is the downside if a docker-compose file is provided with required config? It is not forcing anyone to use docker. And makes it easy to run this with a single command, instead of opening 3 cmd   separately (and 4th one to run the app itself)

(opentelemetry-demo is not the alternate. it is too much for just .NET specific)

Reply:
I think docker-compose is more like a demo (it just works without the reader following steps and understanding what's happening) instead of tutorial (which provides instructions for the users to follow with the intention that they understand how things work after finishing the steps).

Maybe the docker files should be moved to the ./Examples folder.

Reply:
> Maybe the docker files should be moved to the ./Examples folder._

That is fine by me. (If one wants to see exemplars in action, it should be possible with one cmd (docker) from this repo - I don't want to ask them to go check the otel-demo project for that.)

Reply:
#5640